package.json

```diff
@@ -450,6 +450,7 @@
     "stylelint-config-standard-scss": "^3.0.0",
     "supertest": "^6.3.3",
     "supertest-as-promised": "^4.0.2",
+    "tap": "^16.3.4",
```

Comment:
Why did we add this dependency? It appears unused, in which case we should remove the `package.json` and `yarn.lock` changes from this PR.